types/framework/entities/user.d.ts

```diff
@@ -216,6 +216,43 @@ declare class User extends Entity<User.Data> {
   protected _onDelete(options: Entity.DeleteOptions, userId: string): void;
 }
 
+/**
+ * A subclass of Set which manages the Token ids which the User has targeted
+ * @internal
+ */
+declare class UserTargets extends Set<Token> {
+  constructor(user: User);
+
+  user: User;
+
+  /**
+   * Return the Token IDs which are user targets
+   */
+  get ids(): string[];
+
+  /**
+   * @override
+   * @remarks
+   * Returns void, but Set<T>.add returns boolean
```

Comment:
So sad, hope this is also changed in 0.8.0, similar to how `Collection` is apparently getting fixed there...